packages/slice-machine/src/components/SideNav/SideNavEnvironmentSelector.tsx

```diff
@@ -18,7 +18,11 @@ import LogoIcon from "@src/icons/LogoIcon";
 import * as styles from "./SideNavEnvironmentSelector.css";
 
 type SideNavEnvironmentSelectorProps = {
-  variant?: "default" | "unauthorized";
+  variant?:
+    | "default"
+    | "offline"
+    | "UnauthorizedError"
+    | "UnauthenticatedError";
```

Comment:
You are right. I updated the code.

Only problem is that it should not even be like that directly. In order to release we did a quick fix with @hypervillain.

In summary, if you check what has changed since your code, you will see that we don't throw from the manager anymore but return an object with the error and also that we don't use the manager function to check what kind of error it is. Both usage were not working. But for me there is something to improve here. The way we are checking the error is bad, and it's tech debt already.

The problem was only happening in production with an alpha, there is no problem in dev.

I'm talking these two commits:
- https://github.com/prismicio/slice-machine/pull/1217/commits/fa4b88cf1eb890625159955fa3a1ce2fd5a10144
- https://github.com/prismicio/slice-machine/pull/1217/commits/dcfd59c8c9754f6c0d64dc67a4b74dbafc304974

If you have an idea of the problem and how we could improve that part, it would be great 🙂